### DISCUSSION TITLE
Prodotti Venduti in Power BI

### DISCUSSION THREAD

**Discussion by antoniograzioli** [fattureincloud/api#440] [Q&A] [answered]
ciao, sto collegando Power BI e FattureInCloud per un cliente.
Seguendo i vostri esempi e tutorial per Power Query mi sono scaricato tutte le fatture, paginando i risultati, tutto funziona benone e mi scarica oltre 1500 fatture del cliente.
Nella fattura trovo la "**entity**", con i dati del cliente, e finqui bene...
Mi aspettavo di trovare anche una lista di prodotti venduti, ma non capisco dove vengano salvati, se nella **invoice** o in un altro tipo di documento (non mi pare di trovare "_righe_" di fattura...

**Comment by valmoz** [comment, marked as answer]
Buongiorno @antoniograzioli ,
come può vedere dalla nostra [API Reference](https://developers.fattureincloud.it/api-reference/#get-/c/-company_id-/issued_documents) nella risposta è presente il campo **items_list**, contenente l'elenco dei prodotti venduti.

Le consiglio anche di controllare la pagina di [personalizzazione della risposta](https://developers.fattureincloud.it/docs/basics/additional-capabilities/customize-response/), perché alcuni campi potrebbero essere omessi per limitare la dimensione delle risposte.

Ci faccia sapere.
A disposizione
Mauro

**Reply by valmoz** [reply]
Buongiorno @antoniograzioli è riuscito a risolvere?

**Reply by antoniograzioli** [reply]
mi scuso per non aver risposto in tempo utile. 
Sì, usando fieldset=detaild ottengo anche la items_list per avere i prodotti venduti. Grazie
